mock/src/main/java/feign/mock/MockClient.java

```diff
@@ -68,6 +68,7 @@ public synchronized Response execute(Request request, Request.Options options)
     } else {
       responseBuilder = executeAny(request, requestKey);
     }
+    responseBuilder.protocolVersion("mock");
```

Comment:
really liked this idea of setting a different protocol for mock

Reply:
BTW, call it `MOCK` to match HTTP case?!

Reply:
done